action.yml

```diff
@@ -20,6 +20,7 @@ inputs:
       The inputs have been normalized to use kebab-case.
       Use `repository-url` instead.
     required: false
+    default: https://pypi.org/legacy/
```

Comment:
@woodruffw apparently, this caused a regression because PyPI itself has a special-cased upload URL which I forgot about — https://github.com/pypa/gh-action-pypi-publish/issues/130.

Reply:
(fixed in v1.8.1)

Reply:
Good to know, thanks for fixing and sorry for the regression 🙂

Reply:
No worries, it was my oversight...